k8s.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: prebid-server
+  spec:
+    replicas: 4
+    template:
+      metadata:
+        labels:
+          app: prebid-server
+          type: microservice
+      spec:
+        containers:
+        - name: prebid-server
+          image: eu.gcr.io/spil-infra-registry/prebid-server:0000000000
+          ports:
+          - containerPort: 5000
+          resources:
+            requests:
+              cpu: 100m
+              memory: 600Mi
+          livenessProbe:
+            httpGet:
+              path: /prebid-server/static/
+              port: 5000
+            initialDelaySeconds: 10
+            periodSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /prebid-server/static/
+              port: 5000
+            initialDelaySeconds: 10
+            periodSeconds: 20
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: prebid-server
+    labels:
+      app: prebid-server
+      type: microservice
+    annotations:
+      serviceParams: '{
+        "apiVersions": ["v1"]
```

Comment:
you need "something else": prefix 
LB will also not work (because only based on /vN/
you might need LB changes as well, first need to figure out your path-structure